packages/labs/scoped-registry-mixin/src/test/scoped-registry-mixin_test.ts

```diff
@@ -74,7 +72,10 @@ customElements.define('scoped-component', ScopedComponent);
     assert.exists(registry);
   });
 
-  test(`hosted element should not have a registry`, async () => {
+  // In latest spec, the hosted element's registry is window.customElements
+  // but this is not the case in the previous spec implemented in older
+  // versions of the polyfill.
+  test.skip(`hosted element should not have a registry`, async () => {
```

Comment:
delete the test?

Reply:
Removed and cleaned up the tests slightly